boranga/frontend/boranga/src/components/common/species_communities/species_combine/species_combine_profile.vue

```diff
@@ -214,27 +214,31 @@
             <div v-show="!isFauna">
                 <div class="row mb-3" v-for="(species, index) in original_species_combine_list">
                     <label for="" class="col-sm-3 control-label">{{ species.species_number }} Flowering Period:</label>
-                    <div class="col-sm-8">
-                        <select :disabled="true" class="form-select" 
-                            v-model="species.conservation_attributes.flowering_period_id">
-                            <option v-for="option in flowering_period_list" :value="option.id" v-bind:key="option.id">
-                                {{ option.name }}                            
+                    <div class="col-sm-8" :id="select_flowering_period_readonly">
+                        <select :disabled="true" 
+                            style="width:100%;" class="form-select input-sm" multiple 
+                            ref="flowering_period_select_readonly" 
+                            v-model="species.conservation_attributes.flowering_period" >
+                            <option v-for="option in period_list" :value="option.id" :key="option.name">
+                                {{option.name}}
                             </option>
                         </select>
                     </div>
                     <div class="col-sm-1">
                         <input class="form-check-input" type="checkbox" name="flowering_prd_chk" :id="'flowering_prd_chk'+species.id" 
-                        @change="checkConservationInput('flowering_prd_chk','flowering_prd_chk'+species.id,'flowering_period_id',species.conservation_attributes.flowering_period_id)" />
+                        @change="checkConservationInput('flowering_prd_chk','flowering_prd_chk'+species.id,'flowering_period',species.conservation_attributes.flowering_period, 'flowering_period_select')" />
```

Comment:
Plz remove extra parameter at last to the function checkConservationInput()

Reply:
I have added that and can't remove that as it is used for the select2 inputs for the split and combine